File: onlinebanking-facade/src/main/java/domain/BankLoginSettings.java
package domain;

import lombok.Data;

import java.util.HashMap;
import java.util.List;

/**
 * Created by alexg on 14.07.17.
 */
@Data
public class BankLoginSettings {

    private String bank_name;
    private boolean supported;
    private String icon;
    private HashMap<String, String> additional_icons;
    private List<BankLoginCredential> credentials;
    private String auth_type;
    private String advice;
}

File: multibanking-app/src/api/BankLoginSettings.ts
import { BankLoginCredential } from "./BankLoginCredential";

export interface BankLoginSettings {
    "additionalIcons"?: { [key: string]: string; };
    "advice"?: string;
    "authType"?: string;
    "credentials"?: Array<BankLoginCredential>;
    "icon"?: string;
}

